### ISSUE TITLE
IE11 :: Any Modal component :: content overflows

### ISSUE THREAD

**Issue by elbrus** [IBM/carbon-components-react#1889]
## Detailed description

> Describe in detail the issue you're having.

Only in IE11 if you have enough components (or text) rendered inside modal body so that scroll bar should appear, then somehow it doesn't appear and content overflows.

> Is this issue related to a specific component?

Modal, ModalWrapper, ComposedModal

> What browser are you working in?

IE11

> What version of the Carbon Design System are you using?

v9

## Steps to reproduce the issue

1. Go to http://react.carbondesignsystem.com/
2. Go to 1st ModalWrapper example `transactional/passive modal`
3. Launch modal

## Additional information

<img width="1680" alt="screenshot 2019-02-18 at 16 27 21" src="https://user-images.githubusercontent.com/2395165/52960165-d8682a80-33a0-11e9-8b1e-4051bf614285.png">

p.s. I didn't check is it carbon-components-react or carbon-components issue

**Comment by emyarod** [comment]
I'm not able to reproduce this issue on IE11 on Windows 10

![image](https://user-images.githubusercontent.com/8265238/52961017-a08cc300-335f-11e9-85b4-a3a0337348bc.png)

**Comment by elbrus** [comment]
@emyarod decrease top space, I think you have big enough screen so that scroll bar doesn't appear at all
check my provided screen shot, I have bottom (story) part increased

**Comment by emyarod** [comment]
yeah @elbrus I tried that as well, and it still renders properly on my end. Are you able to reproduce this issue in a [Code Sandbox](https://codesandbox.io/s/github/IBM/carbon-components-react/tree/master/examples/codesandbox) too?

![image](https://user-images.githubusercontent.com/8265238/52961692-24937a80-3361-11e9-9c7b-3b4854921120.png)

**Comment by elbrus** [comment]
unfortunately code sandbox with carbon components react doesn't work in IE11 because of this bug: https://github.com/CompuIves/codesandbox-client/issues/1187

**Comment by elbrus** [comment]
@emyarod just to be clear, are you using IE11 or IE Edge with compatibility mode for IE11? Because in 2nd case it works just fine

**Comment by emyarod** [comment]
I just realized my screenshot shows Edge with compatibility mode but I was testing with both @elbrus

![image](https://user-images.githubusercontent.com/8265238/52966814-d5a01200-336d-11e9-9083-578c82aeb40a.png)

**Comment by emyarod** [comment]
I did notice that it overflows momentarily before properly scrolling though, so I will look into this more
